lnrpc/rpc.proto

```diff
@@ -498,7 +500,9 @@ service Lightning {
     rpc AbandonChannel (AbandonChannelRequest)
         returns (AbandonChannelResponse) {
         option (google.api.http) = {
-            delete: "/v1/channels/abandon/{channel_point.funding_txid_str}/{channel_point.output_index}"
+            delete:
+                "/v1/channels/abandon/{channel_point.funding_txid_str}/"
+                "{channel_point.output_index}"
```

Comment:
These changes can be reverted, probably caused by a different version of clang

Reply:
Have reverted the changes, which version of clang are you using?